tests/end2end/features/misc.feature

```diff
@@ -118,6 +118,26 @@ Feature: Various utility commands.
         Then the javascript message "Hello from the page!" should be logged
         And "No output or error" should be logged
 
+    @qtwebkit_skip @qt>=5.11.2
```

Comment:
Here I just copied things from the rest of feature file together since I wasn't familiar with this system.
Even when removing the `@qtwebkit_skip @qt>=5.11.2` and giving them a wrong `Then` no failures were shown. There must be something wrong with this still